.github/workflows/publish.yml

```diff
@@ -2,6 +2,7 @@ name: Deploy docs
 
 on:
   workflow_dispatch:
+  merge_group:
```

Comment:
I don't understand: for the time being, we have to update the `latest` docs while merging, or am I missing something ?

Reply:
This is happening when a PR is added to the merge queue (so it is not merged, but queued and waiting to be merged), not when it is actually merged in `master`, for which another `push` event should be generated (I hope).

If a `push` event is not generated, then we'll have to listen for another kind of event (like [`pull_request: [closed]`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)), but let's worry about it only when and if it will happen.